Tests/Unit/MSFT_xWindowsFeature.Tests.ps1

```diff
@@ -1,205 +1,215 @@
 # These tests use a mock server module. They will fail on an actual server.
 # All tests that require a credential will be skipped
 
-Initialize-TestEnvironment `
-    -DSCModuleName 'xPSDesiredStateConfiguration' `
-    -DSCResourceName 'MSFT_xWindowsFeature' `
-    -TestType Unit `
-    | Out-Null
+Import-Module -Name (Join-Path -Path (Split-Path $PSScriptRoot -Parent) `
```

Comment:
This is less readable than before, so I wouldn't call it a style fix. Could you explain why you chose to rewrite `Import-Module $PSScriptRoot\..\CommonTestHelper.psm1 -Force` (and the other Import-Module statements)? To me it's less readable than before and doesn't seem to provide any benefits.